### DISCUSSION TITLE
Rewrite to Astro

### DISCUSSION THREAD

**Discussion by wanoo21** [timlrx/tailwind-nextjs-starter-blog#710] [Ideas]
Hey Team, I have an idea to rewrite this awesome template for Astro.

What are your thoughts on this?

**Comment by timlrx** [comment]
Sure, be my guest. Happy to link to it for those that might prefer an astro alternative.

**Reply by wanoo21** [reply]
Yay, great, this is what I wanted to hear 😎

**Comment by wanoo21** [comment]
Pushed the first commits, here's the repository- https://github.com/wanoo21/tailwind-astro-starting-blog

Not everything is ready, but I hope in 2-3 days I'll manage to have a working solution with everything ready to get started!

**Comment by wanoo21** [comment]
Hi @timlrx, would you mind including the repo link in the readme? There could be someone who would choose an Astro option.

**Reply by timlrx** [reply]
Most certainly! Sorry didn't notice the previous message as it was buried in the Github discussions. I have added a link to it. How's the experience porting it over? I see that there's no contentlayer support and you have opted to use the native mdx astro plugin?

**Reply by wanoo21** [reply]
Thanks for including the link! Yes, I want to stick more to standard solutions, that are officially supported by Astro. As of now, [contentlayer](https://contentlayer.dev/docs/environments/astro-d57135c9) is just considering adding support for Astro, but nothing official yet.

Looking forward to add support of it as well, but until then, there're many things to be added/improved ;)

**Comment by SangeetAgarwal** [comment]
Hello @timlrx, I did something similar to @wanoo21. I ported you app to the remix platform. The source code is at https://github.com/SangeetAgarwal/tailwind-remix-run-mdxjs-typescript-starter-blog

Though it is mentioned in your [readme](https://github.com/timlrx/tailwind-nextjs-starter-blog), it is buried under Examples V1 (I had initially submitted a pull request for its addition and mentioned it under Examples V1).

Just like the mention for the Astro re-platform, would you be comfortable mentioning it (say) right below "Looking for Astro alternative...".
The blurb could read as "Looking for a Remix-run alternative? Check out the unofficial [Tailwind Remix-run Starter Blog](https://github.com/SangeetAgarwal/tailwind-remix-run-mdxjs-typescript-starter-blog).". If you are ok with mentioning it then I can submit a pull request for the readme.

It has got a few stars and hoping it might help others too. 

Thanks!

**Reply by timlrx** [reply]
Yeah sure, I don't mean to pick favourites 😄

**Reply by SangeetAgarwal** [reply]
Thanks @timlrx. Created a [pull request](https://github.com/timlrx/tailwind-nextjs-starter-blog/pull/743).
